system/CodeIgniter.php

```diff
@@ -1057,7 +1061,7 @@ public function spoofRequestMethod()
      */
     protected function sendResponse()
```

Comment:
Can we deprecate this method since now it is literally just "respond send"?

Reply:
I think it is better the CodeIgniter class has the method, this level of abstraction.